src/scene/gsplat/gsplat.js

```diff
@@ -149,7 +149,7 @@ class GSplat {
         if (preferHighPrecision) {
             halfFormat = false;
         } else {
-            halfFormat = device.extTextureHalfFloat;
+            halfFormat = true;
```

Comment:
Can't you just do `halfFormat = !preferHighPrecision;`?

Reply:
Or just `return !preferHighPrecision;` 😄

Reply:
yep. I was trying to avoid changing the code much as Donovan is in the middle of it .. but have just removed this function now, thanks.